tests/wpt/runner/expectation.json

```diff
@@ -7153,8 +7153,12 @@
       "response": {
         "json.any.html": true,
         "json.any.worker.html": true,
-        "response-blob-realm.any.html": true,
-        "response-blob-realm.any.worker.html": true,
+        "response-blob-realm.any.html": [
+          "realm of the Uint8Array from Response bytes()"
+        ],
+        "response-blob-realm.any.worker.html": [
+          "realm of the Uint8Array from Response bytes()"
+        ],
```

Comment:
These look funky. What are these tests doing?

Reply:
https://github.com/web-platform-tests/wpt/blob/master/fetch/api/response/response-blob-realm.any.js

Reply:
Seems fine since we don't support `iframe`.